.github/workflows/test-build-deploy.yml

```diff
@@ -18,6 +18,11 @@ jobs:
       RACK_ENV: test
       RAILS_ENV: test
       DATABASE_URL: "postgresql://postgres:postgres@127.0.0.1/laa-apply-for-criminal-legal-aid-test"
+      BC_LSC_SERVICE_NAME: ${{ secrets.BC_LSC_SERVICE_NAME }}
```

Comment:
Please don't do this. Tests should not be using secrets at all. And definitely we don't want to add these secrets to Github.
If the ENV variables are needed (ideally they should not, for instance, in code we can do `ENV.fetch("BC_LSC_SERVICE_NAME", nil)`) let's use dummy values.